### ISSUE TITLE
transfer_to_memory should have a ProgressMeter

### ISSUE THREAD

**Issue by kain88-de** [MDAnalysis/mdanalysis#1028]
### Expected behaviour

This should report progress of loading trajectory to memory. This is nice feedback for large trajectories (1GB xtc file for example)

``` python
u.transfer_to_memory(quiet=False)
```
### Actual behaviour

The kwarg isn't accepted.
### How to fix

Add the `ProgressMeter` from `MDAnalysis.lib.log` to the loop reading the coordinates.
### Currently version of MDAnalysis:

(run `python -c "import MDAnalysis as mda; print(mda.__version__)"`)
0.16.0dev0

**Comment by shanmbic** [comment]
Hi @kain88-de  , I tried to implement the ProgressMeter in the function. Could you please review it and suggest what more needs to be added ?Also, should i define the ProgressMeter at the bottom of the function rather than the top ? PR : https://github.com/MDAnalysis/mdanalysis/pull/1054

**Comment by shanmbic** [comment]
Hi @kain88-de , So after defining the ProgressMeter 'pm',  basically should I add a loop , looping over the trajectory object , something like this : 
for ts in u.trajectory[start:stop:step]:
         pm.echo(ts.frame)

and after this loop the trajectory object would be overwritten by the MemoryReader object ? Thanks

**Comment by kain88-de** [comment]
Yes you have to transform the loop comprehension in [line 5076](https://github.com/MDAnalysis/mdanalysis/blob/develop/package/MDAnalysis/core/AtomGroup.py#L5076) into an explicit loop. Just remember create a empty list before the loop and then append to the list in the loop.

**Comment by shanmbic** [comment]
Hey @kain88-de Please review PR: https://github.com/MDAnalysis/mdanalysis/pull/1055
